src/earn/EarnConfirmationScreen.tsx

```diff
@@ -68,23 +69,25 @@ export default function EarnCollectScreen({ route }: Props) {
 
   const feeCurrencies = useSelector((state) => feeCurrenciesSelector(state, depositToken.networkId))
 
+  const withdrawAmountInDepositToken = useMemo(
+    () => inputAmount ?? getEarnPositionBalanceValues({ pool }).poolBalanceInDepositToken,
```

Comment:
nit: so that withdrawAmountInDepositToken's type is always BigNumber
```suggestion
    () => inputAmount ? new BigNumber(inputAmount) : getEarnPositionBalanceValues({ pool }).poolBalanceInDepositToken,
```

Reply:
Changed in 4e205caa242a5421000c24a28cec3578861257e3!